### PR TITLE
Simplify experiments creation and evaluation

### PR DESCRIPTION
**Pull request by RomanSraj**
- Update ReadMe
- Add job_id to experiment run
- Update logging_output_scripts to directly use mlflow data

### CODE REVIEW

**Comment by RomanSraj** [comment]
@heidmic all scripts work as expected. Please have a look at the readme and let me know if there is something unclear or missing. If not, please approve the PR

**Comment by heidmic** [comment]
@RomanSraj can we merge the fixes from https://github.com/heidmic/suprb-experimentation/tree/experiments_working into this PR and merge?

**Comment by RomanSraj** [comment]
> @RomanSraj can we merge the fixes from https://github.com/heidmic/suprb-experimentation/tree/experiments_working into this PR and merge into master?

Yes we can, although the state of these PRs is in no way clean

**Comment by heidmic** [comment]
Alright, then I will wait for your "clean" signal

File: experiments/mlflow.py [hunk 1]
```diff
@@ -68,6 +68,36 @@ def _log_experiment(experiment: Experiment, parent_name: str, depth: int) -> dic
                 result = _log_experiment(subexperiment, parent_name=run_name, depth=depth + 1)
                 if result is not None:
                     results.append(result)
+                if "Decisioasdn Tree" in experiment.name:
```
Comment (heidmic) [inline experiments/mlflow.py:71]: why is there a weird typo?

additionally, I don't think that we, theoretically, want this case in the master to make the code more readable, however, might be the more practical choice

File: experiments/mlflow.py [hunk 2]
```diff
@@ -86,6 +116,34 @@ def _log_experiment(experiment: Experiment, parent_name: str, depth: int) -> dic
 
                         mlflow.set_tag('fold', True)
 
+                        if "Decisioasdn Tree" in experiment.name:
```
Comment (heidmic) [inline experiments/mlflow.py:119]: same as above

File: experiments/mlflow.py [hunk 3]
```diff
@@ -146,6 +204,34 @@ def log_run(estimator: BaseEstimator):
             for step, value in values.items():
                 mlflow.log_metric(key=key, value=value, step=step)
 
+    n_nodes = estimator.tree_.node_count
```
Comment (heidmic) [inline experiments/mlflow.py:207]: is the if missing here?

File: logging_output_scripts/config.json [hunk 4]
```diff
@@ -5,18 +5,14 @@
...
     "heuristics": [
         "ES",
-        "RS",
-        "NS",
-        "MCNS",
-        "NSLC",
-        "NS-P",
-        "MCNS-P",
-        "NSLC-P"
+        "XCSF",
```
Comment (heidmic) [inline logging_output_scripts/config.json:8]: make this two examples rather than replacing the previous example

**Review by heidmic — CHANGES_REQUESTED** [review]

**Comment by heidmic** [comment]
What is missing for merge?

**Comment by RomanSraj** [comment]
Relevant changes are merged

**Pull request closed without merging**